### DISCUSSION TITLE
Will the "Input Labs Alpakka" Open-Source Controller be considered as one of the supported devices for ViGEm 2?

### DISCUSSION THREAD

**Discussion by SecularSteve** [nefarius/ViGEmBus#177] [Ideas]
![alpakka_rainbow](https://inputlabs.io/static/img/alpakka_rainbow.jpg)
![alpakka_black_silver_info](https://inputlabs.io/static/img/alpakka_black_silver_info.jpg)

https://inputlabs.io
https://inputlabs.io/alpakka
https://inputlabs.io/alpakka/gallery

https://github.com/inputlabs
https://github.com/inputlabs/alpakka_firmware
https://github.com/inputlabs/alpakka_pcb
https://github.com/inputlabs/alpakka_case

Very much new, but it seems like a product/project of interest since they have yet to properly implement Wireless Access and Vibration. Not to mention the still lack of properly pre-configured Profiles:

https://github.com/orgs/inputlabs/projects/2
https://inputlabs.io/db/stats

They're welcome to anybody's contributions. Testers included:

https://inputlabs.io/contribute
https://inputlabs.io/blog/hardware_testing

Just thought I should mention this since ViGEm seems like just the kind of application that was made for this.

**Comment by nefarius** [comment]
Not yet since I have no clue what that is and that it exists 😁 I can only add support for devices I own, for reverse engineering. Thanks for the nice summary!
